examples/v1beta1/large-instances.yaml

```diff
@@ -0,0 +1,32 @@
+# This example NodePool will avoid small instance types in the cluster
```

Comment:
(nit) When a YAML file has a `---` line, I put one at the start as well.

That helps people learn that `---` is a document start marker, not a separator.